src/sentry/mail/adapter.py

```diff
@@ -85,9 +84,8 @@ def get_sendable_user_objects(project):
         Return a collection of USERS that are eligible to receive
         notifications for the provided project.
         """
-        return NotificationSetting.objects.get_notification_recipients(project)[
-            ExternalProviders.EMAIL
-        ]
+        recipients = NotificationSetting.objects.get_notification_recipients(project)
+        return set.union(*recipients.values()) if recipients else {}
```

Comment:
some cleeeaan python 🐍

i had to look up almost everything in this line

Reply:
ty stackoverflow